packages/tenant-readmodel-writer/test/converterV1.ts

```diff
@@ -35,6 +35,14 @@ export function toFeatureV1(feature: TenantFeature): TenantFeatureV1 {
         },
       },
     }))
+    .with({ type: "DelegatedProducer" }, (feature) => ({
+      sealedValue: {
+        oneofKind: undefined,
+        producer: {
+          availabilityTimestamp: feature.availabilityTimestamp,
+        },
+      },
+    }))
```

Comment:
```suggestion
    .otherwise(() => {
      throw new Error("Unsupported tenant feature");
    });
```
I agree, since this is only used during tests and this case shouldn't happen I suggest we throw like this

Reply:
Elegant solution 👍🏻 this method should be used only in test context so this change not require additional changes to handle the new error.